Xamarin.Forms.Core.UnitTests/SwipeGestureRecognizerTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace Xamarin.Forms.Core.UnitTests
+{
+    [TestFixture]
+    public class SwipeGestureRecognizerTests : BaseTestFixture
```

Comment:
this is still using spaces

Reply:
Not sure what to change here sorry. When I committed all the code there wasn't an issue with spaces so I'm having trouble resolving these because it looks fine on my end.